.github/workflows/benchmarks.yml

```diff
@@ -109,20 +109,23 @@ jobs:
           path: artifacts
 
       - name: Process data
+        working-directory: scripts/processors/
         shell: pwsh
         run: |
-          Get-ChildItem -File -Path artifacts -Depth 1 | Foreach {. python3 process_results.py $_.fullname data.csv}
+          Get-ChildItem -File -Path artifacts -Depth 1 | Foreach {. python3 process_results.py $_.fullname ../data/nuget.csv}
 
       - name: Generate graph with Plotly for Python
+        working-directory: scripts/processors/
         if: ${{ false }}
         shell: pwsh
         run: |
-          . python3 generate_html.py data.csv index-python.html
+          . python3 generate_html.py ../data/nuget.csv index-python.html
```

Comment:
This step can be removed, it is a remnant from the old system.

Reply:
Alright! ✅🚀